doc/developer-guides/hld/hld-virtio-devices.rst

```diff
@@ -502,7 +507,7 @@ The key data structures for vhost are listed as follows.
 DM APIs
 =======
 
-The DM APIs are exported by DM, and they should be used when realizing
+The DM APIs are exported by DM, and they should be used when configuring
```

Comment:
Same as previously, "configuring" isn't right here, but "implementing" would be OK.